### PR TITLE
Add possibility to have /usr and /var parts on iscsi (#856446)

### PR DESCRIPTION
**Pull request by jkonecny12**
Prevents adding of ``_netdev`` fstab parameter for mountpoints which are on the same LUN where ``/`` mountpoint is. 
This will fix ``/usr`` and ``/var`` are not mounted on boot and ``/boot`` is not mounted in singleuser mode.

We support ``/var`` and ``/usr`` on a LUN only when the ``/`` is on the same LUN.

*Resolves: rhbz#856446*

### CODE REVIEW

File: storage/__init__.py [hunk 1]
```diff
@@ -2424,7 +2426,11 @@ def fstab (self):
                 if device.dependsOn(netdev):
                     if mountpoint == "/":
                         options = options + ",_rnetdev"
-                    else:
+                    # Don't add _netdev to fstab if it is the same network device where root is.
+                    # Or they can wait for network forever, because the mountpoints for
```
Comment (vpodzime) [inline storage/__init__.py]: *mountpoints required for...*

**Comment by vpodzime** [comment]
Looks good to me otherwise.

File: storage/__init__.py (hunk 1 above)
Comment (jkonecny12) [inline storage/__init__.py]: Thank you for this correction. It's fixed now.

**Comment by rvykydal** [comment]
Looks ok to me.
I think this should not break any other valid SAN use cases.

**Comment by jkonecny12** [comment]
Pushed. Thank you guys

**Pull request closed without merging**